### PR TITLE
feat: submission pubsub

### PR DESCRIPTION
**Pull request by bwpikaard**

### CODE REVIEW

**Comment by bwpikaard** [comment]
Do not merge until parent branch is merged to dev.

**Review by 0c370t — DISMISSED** [review]

File: core/src/identity/database/user-authentication-account.repository.ts
Comment (gankoji) [inline core/src/identity/database/user-authentication-account.repository.ts:16]: Do we have a ticket to at least discuss this? Might as well

**Review by gankoji — DISMISSED** [review]

File: core/src/identity/database/user-authentication-account.repository.ts
Comment (bwpikaard) [reply core/src/identity/database/user-authentication-account.repository.ts:16]: Ticketed

**Review by gankoji — APPROVED** [review]

**Pull request merged**
